packages/components/src/box-control/vertical-horizontal-input-controls.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Internal dependencies
+ */
+import UnitControl from './unit-control';
+import { LABELS } from './utils';
+import { Layout } from './styles/box-control-styles';
+
+const groupedSides = [ 'vertical', 'horizontal' ];
+
+export default function VerticalHorizontalInputControls( {
```

Comment:
It seems we could call this `AxialInputControl` to be more succinct and clear?

Reply:
Oh, I like that name! Thanks for the suggestion, I can put up a small PR to rename it 👍

Reply:
Great, thanks!